src/main/java/com/google/collegeplanner/data/Class.java

```diff
@@ -0,0 +1,113 @@
+package com.google.collegeplanner.data;
+
+public class Class {
+  private String courseId;
+  private String name;
+  private String semester;
+  private int credits;
+  private String deptId;
+  private String description;
+  private String[] gradingMethod;
+  private String coreqs;
+  private String prereqs;
+  private String restrictions;
+  private String additionalInfo;
+  private String creditGrantedFor;
+
+  public Class(String courseId, String name, String semester, int credits, String deptId, String description, 
+  String[] gradingMethod, String coreqs, String prereqs, String restrictions, String additionalInfo, String creditGrantedFor) {
+    this.courseId = courseId;
+    this.name = name;
+    this.semester = semester;
+    this.credits = credits;
+    this.deptId = deptId;
+    this.description = description;
+    this.gradingMethod = gradingMethod;
+    this.coreqs = coreqs;
+    this.prereqs = prereqs;
+    this.restrictions = restrictions;
+    this.additionalInfo = additionalInfo;
+    this.creditGrantedFor = creditGrantedFor; 
+  }
```

Comment:
Could you change this to take in a JSONObject for the course, since thats the more common form we'd have this information in.

Reply:
or just make an additional constructor that takes a JSONObject